documentation/modules/oauth/con-oauth-authorization-keycloak-example.adoc

```diff
@@ -3,118 +3,152 @@
 // con-oauth-authorization-keycloak-authorization-services.adoc
 
 [id='con-oauth-authorization-keycloak-example_{context}']
-= Example authorization rules configuration using Authorization Services
+= Example authorization rules configuration using Keycloak Authorization Services
 
-This is an end-to-end example of using Keycloak Authorization Services to configure authorization rules for use with `keycloak` authorization in Strimzi.
-The example starts by deploying the Keycloak server with pre-configured realms, requiring no additional configuration.
-Next, we deploy the Kafka cluster configured to use one of the pre-configured realms.
-We then connect to the Keycloak Admin Console and use the user interface to check the configuration of authorization rules.
-Finally, we use Kafka CLI client tools with different personal and service accounts to demonstrate limiting access based on the permissions granted to different accounts.
+This section provides an end-to-end example of using Keycloak Authorization Services to configure authorization rules for use with `keycloak` authorization.
```

Comment:
To increase the readability, use separate paragraphs for 'First...', 'Second', 'Third' and 'Finally'.